gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -90,6 +87,7 @@
 import org.apache.gobblin.service.monitoring.ResumeFlowEvent;
 import org.apache.gobblin.util.ConfigUtils;
 import org.apache.gobblin.util.reflection.GobblinConstructorUtils;
+import org.eclipse.jetty.util.ConcurrentHashSet;
```

Comment:
not familiar w/ this one... it may be fine, but you could always just use `j.u.c.ConcurrentHashMap` w/ `true` for the value.